index.html

```diff
@@ -1770,6 +1848,28 @@
             default presentation.
           </div>
         </section>
+        <section>
+          <h4>
+            Receiving user agent
+          </h4>
+          <p>
+            The following partial interface MUST be implemented by a
+            <a>receiving user agent</a>:
+          </p>
+          <pre>
+          partial interface Presentation {
+            [SameObject]
+            readonly attribute PresentationReceiver? receiver;
+          };
+          
+</pre>
+          <p>
+            The <dfn for="Presentation"><code>receiver</code></dfn> MUST return
+            <code>null</code> if the current <a>browsing context</a> is not a
+            <a>receiving browsing context</a> and return a
+            <a><code>PresentationReceiver</code></a> instance otherwise.
+          </p>
```

Comment:
Can the current browsing context not be a receiving browsing context for a receiving user agent?

In the end, keeping just one `Presentation` interface might make our lives easier (on top of other things, ReSpec seems to lose itself if we use two different partial definitions). The prose could simply say that:

1. `defaultRequest` will always return null in a receiving user agent that is not a controlling user agent as well.
2. `receiver` will always return null in a controlling user agent that is not a receiving user agent as well.
3. a controlling user agent MUST NOT implement the `PresentationReceiver` interface, even if it appears in the interface definition
4. a receiving user agent MUST NOT implement the `PresentationRequest` interface, even if it appears in the interface definition.

Note I say "must not" above because the existence of these interfaces should mean that they can be used!

What do you think?